src/wp-admin/includes/class-ftp.php

```diff
@@ -797,7 +797,7 @@ function glob_pattern_match($pattern,$subject) {
 		$chunks=explode(';',$pattern);
 		foreach($chunks as $pattern) {
 			$escape=array('$','^','.','{','}','(',')','[',']','|');
-			while(strpos($pattern,'**')!==false)
+			while( str_contains( $pattern, '**' ) )
```

Comment:
Should we remove the spaces or should we stay with `strpos()` in this file?

Reply:
```suggestion
			while(str_contains($pattern,'**') )
```

```suggestion
			while(strpos($pattern,'**')!==false)
```

Reply:
It looks like the library is adopted to I think it's fine to keep the change here. The spaces can remain,